src/components/ChatView.tsx

```diff
@@ -582,6 +585,21 @@ const ChatView = forwardRef<ChatViewHandle, ChatViewProps>(({
                                                             padding: "12px 24px",
                                                             resize: "none"
                                                         }}
+                                                        onCopy={(event) => {
+                                                            if (disableCopyPaste) {
+                                                                event.preventDefault();
+                                                            }
+                                                        }}
+                                                        onPaste={(event) => {
+                                                            if (disableCopyPaste) {
+                                                                event.preventDefault();
```

Comment:
better to show a toast as well saying "copying is disabled for this question" (or pasting or cutting based on the action taken)